W 12217812 credentialusers se (#124)
mulesoft/docs-rpa: rpa-manager/modules/ROOT/pages/processautomation-deploy.adoc

+

[[test-configuration-link-globals]]

> Alternatively, you can set the values of the activity parameters by linking them with a global variable. To do this, click the *Link a global activity parameter* icon image:linkglobalvariable-icon.png[ symbol,1.5%,1.5%]. RPA Manager only shows variables assigned to the test phase and only credentials for which you belong to the xref:processautomation-prepare.adoc#form-create-credential-useraffiliation[affiliated users] for linking. If a credential affiliated to another user is already linked, RPA Manager displays a row of stars to hide the data. The variables are typed. You can only enter or link values of the same type.  The values entered here are only start values and they can be overwritten by other values during the course of the process run.

Reviewer: Mismatch using singular and plural nouns in the above text.

> Alternatively, you can set the values of the activity parameters by linking them with a global variable. To do this, click the *Link a global activity parameter* icon image:linkglobalvariable-icon.png[ symbol,1.5%,1.5%]. RPA Manager only shows variables assigned to the test phase and only credentials for which you belong to the xref:processautomation-prepare.adoc#form-create-credential-useraffiliation[affiliated users] for linking. If a credential affiliated to another user is already linked, RPA Manager displays a row of stars to hide the data. The variables are typed. You can only enter or link values of the same type.  The values entered here are only start values and they can be overwritten by other values during the course of the process run.

Reviewer: ```suggestion
Alternatively, you can set an activity parameter value by linking it to a global variable. To do this, click the *Link a global activity parameter* icon image:linkglobalvariable-icon.png[ symbol,1.5%,1.5%]. RPA Manager shows variables assigned to the test phase and credentials for which you belong to the xref:processautomation-prepare.adoc#form-create-credential-useraffiliation[affiliated users] for linking. If a credential affiliated to another user is already linked, RPA Manager displays a row of stars to hide the data. The variables are typed. You can only enter or link values of the same type.  The values entered here are only start values and they can be overwritten by other values during the course of the process run.
```

Reviewer: This text needs rework - "and credentials for which you belong to the xref:processautomation-prepare.adoc#form-create-credential-useraffiliation[affiliated users] for linking"

I am not sure what it means so I can't help reword it.

Reviewer: I removed some "only"s because I don't think they are needed here.

Author: Global variables and (global) credentials are created in other views. There are many of them. Not all are suitable to be linked to a specific activity parameter (which is initialized in a run configuration (either by keeping the default values defined in RPA Builder, or by entering a value manually, or by linking the value contained in a global variable). 
Therefore RPA Manager only shows suitable variables (and not all of them) for linking (with an activity parameter). In the case of credentials, the credentials have to be suitable AND the user creating the run configuration has to be affiliated.

....

Author: -->
Added the following item to the Before You Begin section:
If you want to use the values of [global variables](https://github.com/mulesoft/docs-rpa/pull/processautomation-prepare.html#manage-globalvariables) or [credentials](https://github.com/mulesoft/docs-rpa/pull/processautomation-prepare.html#manage-credentials) as initial values for activity parameters, an RPA administrator must create them in the Global Variables or Credential Pool view first.

Author: -->
Changed the text for activity parameters in the wizard to:
Alternatively, you can set the value of an activity parameters by linking it with a global variable. To do this, click the *Link a global activity parameter* icon image:linkglobalvariable-icon.png[ symbol,1.5%,1.5%] to select a variable for linking. RPA Manager displays variables that are of the same type and are assigned to the [test, production] phase, and only credentials for which you belong to the xref:processautomation-prepare.adoc#form-create-credential-useraffiliation[affiliated users]. If a credential affiliated to another user is already linked, RPA Manager displays a row of stars to hide the data.

Author: Is that better?

> * *User Task*:

Reviewer: I would remove the colons after each bolded list item. I realize this is older text.

+
When you select a bot, ensure that it is configured for the required session type, that it is allowed to be deployed in the relevant phase, and that it can operate the required applications.
+

> If one of the selected bots is not configured appropriately, a notification appears and you cannot proceed with the configuration.

Reviewer: ```suggestion
If one of the selected bots is not configured appropriately, a notification appears, and you cannot proceed with the configuration.
```

Reviewer: Can you make this more active by saying "RPA displays a notification..."?
